### PR TITLE
Fix remaining typos discovered by codespell

### PR DESCRIPTION
**Pull request by cclauss**
Finish the work started in #208 and lock in the tests.

### CODE REVIEW

File: docs/api/pool.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `!ConnectionPool` class
 .. autoclass:: ConnectionPool
 
    This class implements a connection pool serving `~psycopg.Connection`
-   instances (or subclasses). The constructor has *alot* of arguments, but
+   instances (or subclasses). The constructor has *a lot* of arguments, but
```
Comment (dvarrazzo) [inline docs/api/pool.rst:46]: That's wanted

Comment (cclauss) [reply docs/api/pool.rst:46]: https://www.grammarly.com/blog/a-lot-alot-allot/

Comment (dvarrazzo) [reply docs/api/pool.rst]: https://hyperboleandahalf.blogspot.com/2010/04/alot-is-better-than-you-at-everything.html

Comment (dvarrazzo) [reply docs/api/pool.rst]: I am aware of the grammar. That was for personal amusement :)

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -9,6 +9,16 @@ on:
   pull_request:
 
 jobs:
+  codespell:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (dvarrazzo) [inline .github/workflows/lint.yml:16]: This is very appreciated, thank you very much. I didn't expect to see so many grammar errors... _alot!_ :smile: 

The github action doesn't describe how to run the tool locally. Would it be possible to implement it in `tox.ini` instead, to make the command available to both command line (and pre-commit scripts, for who use them) and use it in the pipeline, by a step:

```yaml
      - name: Check grammar
        run: tox -e codespell
```

like the ones below in this file?

**Review by dvarrazzo — CHANGES_REQUESTED** [review]

**Comment by dvarrazzo** [comment]
Hello @cclauss, I have seen that you are closed this MR. Does it mean you are no more interested in setting up this check in the pipeline, or should we expect you do get a new MR?

Please let us know. Either is fine for me, and I thank you for your work.

Cheers

**Pull request closed without merging**